src/client/tls13.rs

```diff
@@ -826,8 +828,12 @@ impl hs::State for ExpectFinished {
             /* Derive the client-to-server encryption key before key schedule update */
             let key = sess.common
                 .get_key_schedule()
-                .derive_bytes(SecretKind::ClientHandshakeTrafficSecret,
-                              &st.handshake.hash_at_client_recvd_server_hello);
+                .derive_logged_secret(
+                    SecretKind::ClientHandshakeTrafficSecret,
+                    &st.handshake.hash_at_client_recvd_server_hello,
+                    &*sess.config.key_log,
+                    sess.common.protocol.labels().client_handshake_traffic_secret,
+                    &st.handshake.randoms.client);
```

Comment:
With this change, this secret is logged earlier than it was previous. I don't know if this makes any difference.